### ISSUE TITLE
Parallel test failing when CI_NODE_TOTAL is strictly greater than total number of test files

### ISSUE THREAD

**Issue by micaelmbagira** [avajs/ava#2461]
**How to reproduce**

Consider an app with only 1 test file, `index.spec.ts`:

```ts
import test from 'ava';

test('pass', t => {
	t.pass();
});
```
and set the following environment variables

```
export CI_NODE_INDEX=0
export CI_NODE_TOTAL=3
export AVA_FORCE_CI='ci'
```

Run the test

```
$ ava

  ✔ pass

  Ran 1 test file out of 1 for job 1 of 3

  1 test passed

✨  Done in 1.83s.
```

Then change the `CI_NODE_INDEX`: `export CI_NODE_INDEX=1`

**Expected**

ava cannot find any test to run for this index (because there's only 1 test file in total) and **exits with 0**.

**Current behavior**

ava exits with code 1 because it can't find any test file.

```
$ ava

  ✖ Couldn’t find any files to test

error Command failed with exit code 1.
```

The use case is that I have a monorepo and I activate the parallelisation globally at CI level. Some of the monorepo projects have more than 1 file, some don’t.

**Comment by novemberborn** [comment]
I agree, it should recognize this scenario and exit with code 0, and perhaps `✅ No files tested in this parallel run` or something along those lines.

**Comment by micaelmbagira** [comment]
@novemberborn i’ll try to open a PR.

**Comment by novemberborn** [comment]
I've shipped the fix in v3.7.1.

**Comment by spectranaut** [comment]
I just want to say that I just notice this bug and I'm so pleased all I have to do is update ava :) :) thx everyone!
